defs/config_def.py

```diff
@@ -106,7 +107,10 @@
     {NAME: "scroll_unselected", TOOLTIP: ttip.field_widget_scroll_unselected}, # RENAMED
     {NAME: "evaluate_entry_fields", TOOLTIP: ttip.field_widget_evaluate_entry_fields},
     {NAME: "show_structure_meta", TOOLTIP: ttip.field_widget_show_structure_meta},
-    DEFAULT=sum([1<<i for i in (2, 3, 4, 6, 7, 8, 9, 10)])
+    DEFAULT=(
+        sum([1<<i for i in (2, 3, 4, 6, 7, 8)])
+        if e_c.IS_LNX else
+        sum([1<<i for i in (2, 3, 4, 6, 7, 8, 9, 10)]))

```

Comment:
A comment explaining this would be nice.

Reply:
Done. Will be included in update tonight.